### PR TITLE
fix: ignore unlisted buffers

### PR DESCRIPTION
**Pull request by Adventune**

### CODE REVIEW

File: lua/testaustime.lua [hunk 1]
```diff
@@ -35,6 +35,10 @@ function sendheartbeat()
 
     local hb = getheartbeatdata()
 
+    if vim.fn.buflisted(vim.fn.bufnr("%")) == 0 then
+        return
+    end
```
Comment (DrVilepis) [inline lua/testaustime.lua:40]: This should probably use `0` which is a magic id for the current buffer instead of `vim.fn.bufnr("%")`

**Review by DrVilepis — CHANGES_REQUESTED** [review]
Small change then its ready to merge

File: lua/testaustime.lua (hunk 1 above)
Comment (Adventune) [reply lua/testaustime.lua:40]: Doesn't seem to give the same results (atleast on my installation):
1. Open a single buffer
2. Run `:lua vim.fn.buflisted(vim.fn.bufnr("%"))` vs run `:lua vim.fn.buflisted(0)`
3. Open another buffer and both give the same result.

Comment (DrVilepis) [reply lua/testaustime.lua:40]: aamunen case, merging

**Pull request merged**